### PR TITLE
fix few typos/inconsistencies in latvian translations

### PR DESCRIPTION
**Pull request by k0d3r1s**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| License       | MIT

Found few typos and few inconsistencies in Latvian translations.

Comment for inconsistencies:  
```"ieeja" and "autentifikācija" are synonyms in Latvian but as "autentifikācija" is used in more often in other translation keys, this change makes it less confusing```

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.lv.xlf [hunk 1]
```diff
@@ -64,11 +64,11 @@
             </trans-unit>
             <trans-unit id="17">
                 <source>Too many failed login attempts, please try again later.</source>
-                <target>Pārāk daudz atteiktu ieejas mēģinājumu, lūdzu, mēģiniet vēlreiz vēlāk.</target>
+                <target>Pārāk daudz atteiktu autentifikācijas mēģinājumu, lūdzu, mēģiniet vēlreiz vēlāk.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Security/Core/Resources/translations/security.lv.xlf:67]: Google Translate says that your proposed change means:

```
Too many rejected authentication attempts, please try again later
```

But the original meant:

```
Too many rejected login attempts, please try again later.
```

which is exactly like the original.

I'm not saying that you are wrong, but please double check that changing "login attempt" by "authentication attempt" is the right change here. Thanks.

Same for the change of _"Ieejas"_ by _"Autentifikācijas"_ in the other message.

Comment (k0d3r1s) [reply src/Symfony/Component/Security/Core/Resources/translations/security.lv.xlf:67]: "ieejas" in this context is wrong word as you are not phisically going in somewhere. I chose "autentifikācijas" as it was used for "login" in other keys. Technically best word here for "login" is "pieslēgties" but then change must be made in more translation keys. I can do that later

Comment (javiereguiluz) [reply src/Symfony/Component/Security/Core/Resources/translations/security.lv.xlf:67]: OK, so if you double checked this, then it's OK. Thanks for the explanation.

**Review by javiereguiluz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @k0d3r1s.

**Pull request merged**